### DISCUSSION TITLE
Maybe `Iden` should be renamed to `Ident`?

### DISCUSSION THREAD

**Discussion by liigo** [SeaQL/sea-query#534] [Q&A]
`Ident` is much more meaningful than `Iden`. `Ident` is googleable.

**Comment by billy1624** [comment]
Hey @liigo, thanks for asking!! It would be too bad for us to rename it at this point.

The `t` at the end of `Ident` is silence so I guess that's why we get rid of it in the first place and for naming shorter the better. That's why we ended up with `Iden`.

The documentation is there to explain `Iden` to newcomers. https://github.com/SeaQL/sea-query#iden

**Reply by liigo** [reply]
sorry but I still don't understand why you accept an un-googleable abbreviation of `identification`. and why you can't change it? The current version of sea-query is 0.27.2, not a stable version. `Ident` and `Identification` is both better than `Iden`.
